Sources/SwiftUIPager/PagerContent+Buildable.swift

```diff
@@ -178,6 +178,10 @@ extension Pager.PagerContent: Buildable {
         mutating(keyPath: \.alignment, value: value)
     }
 
+    func forwardOnly(_ value: Bool) -> Self {
```

Comment:
do you mind add the same `Swift Docs` as in `Pager+Buildable.swift` please 🙏 ?

Reply:
I think adding the parameter to the swift doc comment did this, though let me know if you think I'm still missing something

Reply:
For the rebase I think I'm already in scyn, When I look at sync with fork in GitHub I get No new commits to fetch. Enjoy your day!

Reply:
My issue was I'd branched from main not develop, easiest way to fix this was to branch off develop and put up another  pull request https://github.com/fermoya/SwiftUIPager/pull/264. Please delete this pull request the new pull request should hopefully have all the changes required